File: Python/완주하지 못한 선수.py
def solution(participant, completion):
    answer = ''
    player = {}

    for i in participant:
        if i in player:
            player[i] += 1
        else:
            player[i] = 1

    for i in completion:
        player[i] -= 1

    for key, value in player.items():
        if value > 0:
            answer = key
            break

    return answer

File: JavaScript/완주하지 못한 선수.js
function solution(participant, completion) {
    var dict = {};

    for (var i in participant) {
        var person = participant[i]
        if (person in dict) {
            dict[person] += 1;
        } else {
            dict[person] = 1;
        }
    }

    for (var i in completion) {
        var person = completion[i]
        dict[person] -= 1;
    }

    for (var i in dict) {
        if (dict[i] == 1) {
            return i;
        }
    }

    return -1;
}